gordon/resources/s3.py

```diff
@@ -265,13 +265,13 @@ def __init__(self, *args, **kwargs):
 
     def get_bucket_arn(self):
         bucket = self.settings.get('bucket')
-        if bucket.startswith('arn:aws:'):
+        if isinstance(bucket, str) and bucket.startswith('arn:aws:'):
```

Comment:
Other spots in s3.py seems to prefer `isinstance(destination, six.string_types):` over just checking `str`. 

https://github.com/jorgebastida/gordon/blob/master/gordon/resources/s3.py#L127